deprecated/raidrive-aoks.json

```diff
@@ -17,10 +17,10 @@
         ]
     ],
     "checkver": {
-        "url": "https://www.raidrive.com/download"
+        "url": "https://www.raidrive.com/download",
```

Comment:
manifest is broken ;) anyhow, but thanks, fixed this now... just in case, you never know :)

Reply:
I'm sry what is the problem here? juts a whitespace fix, I didn't even make it, it was auto formatted